### ISSUE TITLE
Complete example ?

### ISSUE THREAD

**Issue by jbrodriguez** [AriaFallah/mobx-store#49]
Could you share the code for the demo ?

**Comment by AriaFallah** [comment]
@jbrodriguez 

The demo was made a while ago, and the library is currently in a really weird place...I'll keep this issue open and get back to you when I have something presentable.

**Comment by jbrodriguez** [comment]
@AriaFallah sure, I realized later that something was off when I saw that "destroy store concept" commit :)

**Comment by AriaFallah** [comment]
@jbrodriguez 

Yeah when I made the very first iteration of the library there was no `spy` API, and MobX had just come out. The only way to do anything I wanted was to couple everything to a `store` object.

Once I switched to using `spy` to enable deep observation for undo/redo, I realized that the whole `store` construct was completely unnecessary, and I began moving away from it. Now all you're left with is `undo`, `redo`, and `chain` as utilities.

Not sure what to even call the library now.
